fuzy-norms/script.whizzml

```diff
@@ -0,0 +1,232 @@
+;; T-norm and T-conorm operators implemented in flatline and whizzml
+(define (check-input fd)
+  (when (not (numeric-field? fd))
+    (raise {"message" (str "T-norm or T-conorm operand must be numeric: "
+                           (fd "id"))
+            "code" -2}))
+  (when (or (< (get-in fd ["summary" "minimum"]) 0)
+            (> (get-in fd ["summary" "maximum"]) 1))
```

Comment:
A nicer and more compact way to write this is:
      (when (not (<= 0 (fd ["summary" "maximum"]) 1))